src/Esprima/Ast/LogicalExpression.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Esprima.Ast
```

Comment:
nit: could use file-scoped namespace directly for new code

Reply:
Currently namespace style is completely mixed in the codebase. Let's address this with another PR (there's an .editorconfig switch for this).

Reply:
The problem with existing code is that if changed, every line will be changed in files and history (blame) will be hard...